README.md

```diff
@@ -78,6 +78,13 @@ I'm open to adding more drivers, feel free to file an issue.
 
 ## To use from Kubernetes:
 
+### Using Deployment and Service
+Follow this [instruction](https://github.com/GoogleCloudPlatform/cloudsql-proxy/blob/master/Kubernetes.md)
+
+### Using Helm
+Follow this [instruction](https://github.com/kubernetes/charts/tree/master/stable/gcloud-sqlproxy)
```

Comment:
Looking through this page I see that the tag of the newest version of the Proxy is already out of sync (1.10 is already released). How should we make sure that page stays up to date?